src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  &--0 {
```

Comment:
it should work not only for 0

it should change number of active stars as in the previous task